tests/js/common/shell/shell-pregel-start.js

```diff
@@ -0,0 +1,116 @@
+/*jshint globalstrict:false, strict:false */
```

Comment:
I think this file should be named `-cluster.js`, as it is supposed to be executed only in cluster mode, right?